### PR TITLE
Update workflow to add new dependencies

### PR DESCRIPTION
**Pull request by amoralej**
According to new automation based on rdoinfo.

### CODE REVIEW

File: source/documentation/requirements.html.md [hunk 1]
```diff
@@ -80,42 +83,106 @@ the CloudSIG using rdopkg:
...
+    CBS tags applied to each package (shown under buildsys-tags section for each package).
+    Tags have a format cloud7-openstack-&lt;release>-&lt;phase> where:
+
+    - release: is a the OpenStack release name, as pike, queens or rocky.
+    - phase:
+      - `candidate` phase is assigned to packages to be rebuilt in
+      CBS but not pushed to any RDO repository.
+      - `el7-build` (only available for Rocky and newer releases)
+      is assigned to packages that only required to build other
+      packages but not to deploy or run any OpenStack service.
```
Comment (javierpena) [inline source/documentation/requirements.html.md]: but are not a runtime requirement for any other package (it looks a bit confusing).

File: source/documentation/requirements.html.md [hunk 2]
```diff
@@ -80,42 +83,106 @@ the CloudSIG using rdopkg:
...
+
+    Note that, for the release currently under development (rocky right now), only the
+    testing phase will be available. The package included in cloud7-openstack-rocky-testing
+    will be the one used to deploy from RDO Trunk Master repositories and it will be
+    automatically pushed to cloud7-openstack-rocky-release at RDO Rocky is officially
+    released and published.
+
+    If the package is found for the required CBS tag, it's already in RDO repositories
+    and no more actions are needed to add it to the repos.
+
```
Comment (javierpena) [inline source/documentation/requirements.html.md]: We may want to add a simple flow to explain the flow: 

-candidate -> -testing -> -release
-candidate -> -el7-build

File: source/documentation/requirements.html.md [hunk 3]
```diff
@@ -80,42 +83,106 @@ the CloudSIG using rdopkg:
...
 packager needs to participate in this process. While RDO core members may maintain the new package for
 common requirements used by different projects, dependencies for specific project must be maintained in
-Fedora by the project team. Once the package is included in Fedora repos you can create a bugzilla as
-explained in step 5.
-    
+Fedora by the project team. Once the package is included in Fedora repos you can
+create a gerrit review as explained in step 5.
+
+    <br />
+7. Once the package is rebuilt in CBS (review in step 5 is merged) you can pushed
```
Comment (javierpena) [inline source/documentation/requirements.html.md]: you can push it  (typo in pushed)

File: source/documentation/requirements.html.md (hunk 2 above)
Comment (amoralej) [reply source/documentation/requirements.html.md]: It's kind of explained in the workflow https://github.com/amoralej/website/blob/d1bde77d996e0aa390db357e5cd95b234a1f0088/source/images/new-dependencies.png

File: source/documentation/requirements.html.md (hunk 1 above)
Comment (amoralej) [reply source/documentation/requirements.html.md]: fixed

File: source/documentation/requirements.html.md (hunk 2 above)
Comment (amoralej) [reply source/documentation/requirements.html.md]: I've added the flow in second patch.

File: source/documentation/requirements.html.md (hunk 3 above)
Comment (amoralej) [reply source/documentation/requirements.html.md]: fixed

**Comment by hguemar** [comment]
Thanks!

**Pull request merged**